### DISCUSSION TITLE
Can Bromite have similar functionality to NoScript Addon

### DISCUSSION THREAD

**Discussion by spiderman973** [bromite/bromite#1382] [Q&A]
Can Bromite have similar functionality to NoScript Addon (I am not saying that NoScript addon should be added to bromite because it is not possible until chromium upstream supports adding addons)

**Comment by chromer030** [comment]
After merging https://github.com/bromite/bromite/pull/857 , it could be possible.

**Comment by xarantolus** [comment]
If you just want to disable JavaScript, it should be possible in Settings -> Site Settings (under Advanced) -> JavaScript. You can disable it for all sites there. Then if you want to enable /disable it for a specific site only, visit the site, press the lock in the address bar, go to permissions and enable JavaScript. That way it's disabled by default, but you can simply enable it if you want to allow it

**Reply by spiderman973** [reply]
But No Script Addon can block specific domains on sites e.g. If i want to block doubleclick.net on a news website i don't have to block javascript for the news website
